cypress/e2e/laboratory/preflight.cy.ts

```diff
@@ -85,8 +51,22 @@ describe('Preflight Script Modal', () => {
     setMonacoEditorContents('env-editor', env);
   });
 
+  it('code is validated with TypeScript', () => {
+    const tsErrorMessage = "Type 'string' is not assignable to type 'number'.";
+    const script = 'let a = 1; a = ""';
+    cyp.setEditorContent(script);
+    cy.wait(1000); // :(
+    cy.dataCy(s.modal.editorCy)
+      .find('textarea')
+      .focus()
+      // Followed instructions but does not work https://github.com/dmtrKovalenko/cypress-real-events?tab=readme-ov-file#installation
+      // @ts-expect-error
+      .realPress(['Alt', 'F8']);
+    cy.contains(tsErrorMessage);
+  });
```

Comment:
Given the noisy diff caused by refactoring, highlighting that here is the new test for this feature.